tests/unit/GoLoadingElement/GoLoadingElement.spec.js

```diff
@@ -0,0 +1,15 @@
+import { createLocalVue } from '@vue/test-utils'
+import * as AgoUikit from '../../../src'
+import { GoLoadingElement } from '../../../src'
```

Comment:
Can you use `@` or something else ?

Reply:
@Kernes it should be done

Reply:
We can't as the tests are failing if we do this... need to check it w/ Renan